code/modules/clothing/masks/gasmask.dm

```diff
@@ -100,13 +100,13 @@
 	actions_types = list(/datum/action/item_action/toggle_mask, /datum/action/item_action/change_appearance_mask, /datum/action/item_action/toggle_voicechanger)
 	species_fit = list(VOX_SHAPED, GREY_SHAPED)
 	var/list/clothing_choices = list()
-
+//someone who isn't me should really refactor this so it isnt instantiating all the masks in the game on startup
```

Comment:
I knew this but I didn't want to bother digging through voice changer code to see if anything needed actual instances.